### DISCUSSION TITLE
Scraping for Dub anime

### DISCUSSION THREAD

**Discussion by Brendanbam** [itsToggle/plex_debrid#108] [Q&A]
I am trying to find a way to sort and find the english dub version of anime from my jackett sources any ideas?

**Comment by itsToggle** [comment]
Im not sure which indexers work best for anime content, but you should add a sorting rule to plex_debrid that selects for dubbed releases. You do this by adding:
regex: "(dubbed)"
attribute: "title"
Interpretation: "text"
descending: "1"
After adding the rule, move it to the first position for it to have maximum effect.
